File: Problem/344-reverse-string/reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s)
    {
        vector<char> d(s);
        int i = 0;
        while (i < s.size())
        {
            s[i] = d[s.size() - 1 - i];
            i++;
        }
        
    }
};

File: 344-reverse-string/reverse-string.py
class Solution(object):
    def reverseString(self, s):
        """
        :type s: List[str]
        :rtype: None Do not return anything, modify s in-place instead.
        """
        left = 0
        right =len(s)-1

        while right > left :
            tmp = s[right]
            s[right] = s[left]
            s[left] = tmp
            left += 1
            right -= 1
